presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -241,6 +367,14 @@ uint64_t SystemConfig::httpMaxAllocateBytes() const {
   return opt.value_or(kHttpMaxAllocateBytesDefault);
 }
 
+uint64_t SystemConfig::queryMaxMemoryPerNode() const {
```

Comment:
Default to be 4GB? How do we get this number?

Reply:
From Java - it has 0.1 * node memory.
Since we have 40GB default, then 4GB is default here. :)

Reply:
But for us it is overriden by max query memory, right?

Reply:
Yes, in tw config and then in config.properties we have "query.max-memory-per-node" set to 7GB.